Dalamud/Interface/InterfaceManager.cs

```diff
@@ -200,7 +200,7 @@ public TextureWrap LoadImageRaw(byte[] imageData, int width, int height, int num
             return null;
         }
 
-        private IntPtr PresentDetour(IntPtr swapChain, uint syncInterval, uint presentFlags)
+        private unsafe IntPtr PresentDetour(IntPtr swapChain, uint syncInterval, uint presentFlags)
```

Comment:
Do you actually need unsafe anymore?

Reply:
`ImGuiNative.ImFontConfig_ImFontConfig()` returns an `ImFontConfig*` (which is implicitly convertable and converted to an `ImFontConfigPtr`, but still requires `unsafe`).